src/ingest_llm_as/observability/langfuse_client.py

```
@@ -6,207 +6,120 @@
-        """Create a span within an existing trace."""
-        if not self.client:
+
+        try:
+            # Merge input_data into metadata if provided
+            if input_data:
+                metadata = metadata or {}
+                metadata.update(input_data)
+
+            trace = self.client.start_as_current_span(name=name, metadata=metadata)
+            return getattr(trace, "id", None)
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)